yext/studio
# Display external components in AddComponentButton

- Display external components in AddComponentButton
- updated the external component plugin to list exported component using string type, and ComponentExportConfig type to specify initial props

SLAP-2391
TEST=manual

- see that external components like SearchBar and ApplyFiltersButton is displayed on "Add Component" dropdown button
- see that clicking on SearchBar and ApplyFiltersButton correctly add it to the page -- show up on PagePreview and on the component tree on the left.

![Screen Shot 2022-10-03 at 4 09 22 PM](https://user-images.githubusercontent.com/36055303/193669586-a085bc3d-d6b0-4ae7-9f06-0b314fa69be8.png)
